packages/data-stores/src/woocommerce-eligibility/test/reducer.ts

```diff
@@ -0,0 +1,32 @@
+import { transferEligibility } from '../reducer';
+import { TransferEligibility } from '../types';
+
+describe( 'reducer', () => {
+	it( 'returns the correct default state', () => {
+		const state = transferEligibility( undefined, { type: 'TEST_ACTION' } );
+		expect( state ).toEqual( undefined );
+	} );
+
+	it( 'returns the site transfer elibibility', () => {
```

Comment:
Typo here:

```suggestion
	it( 'returns the site transfer eligibility', () => {
```

Also, I'd rephrase this to something like:

```suggestion
	it( 'keys transfer eligibility by site ID', () => {
```